### DISCUSSION TITLE
TL-WN823NV2 slow internet speed Big Sur

### DISCUSSION THREAD

**Discussion by mhristev** [chris1111/Wireless-USB-Big-Sur-Adapter#150] [Q&A]
I recently installed my first hackintosh Big Sur with OpenCore. I don’t have a propper wireless card, so i tried with TL-WN823NV2 (wireless USB adapter). I managed to connect to the WiFi with this applications and everything seemed fine, but when I try to google something - the internet speed is super slow. 
I ran WiFi speed test and the results are : 
Download : 2.83 Mbit/s
Upload : 0.00 Mbit/s 
So it seems like I have connection, but it's very slow. When I ran the test on my phone for example I get much higher results.  

Is it a bug with the adapter or something more? I reinstalled the app multiple times, disabled SecurityBoot and setting the ip manually didn’t help at all. Any help would be appreciated!

**Comment by KernkraftX360** [comment]
I'm using the same wireless adapter, but it didn't work on my open core. Help on how to make it work mate. Are you using the v11?

**Comment by NumanIbnMazid** [comment]
Switching to a different USB port worked for me. Let's give a try.
